### PR TITLE
Add sin operator Jax Frontend - resolving conflicts

### PR DESCRIPTION
**Pull request by karthikgauti**
#2886

### CODE REVIEW

**Comment by karthikgauti** [comment]
Hi @ankitdsi2010, Can you please do code review on this PR?

**Review by ankitdsi2010 — CHANGES_REQUESTED** [review]
Add out parameter in the function and then make the necessary changes in the test

**Comment by karthikgauti** [comment]
> Add out parameter in the function and then make the necessary changes in the test

I don't get what u referred to as I already passed arguments in the function. Can you please elaborate?

**Comment by ankitdsi2010** [comment]
add an additional parameter out as implemented [here](https://github.com/unifyai/ivy/pull/3001/files)

File: ivy/functional/frontends/jax/lax/operators.py [hunk 1]
```diff
@@ -54,3 +54,10 @@ def acos(x):
 
 
 acos.unsupported_dtypes = {"torch": ("float16", "bfloat16")}
+
+
+def sin(x):
```
Comment (ankitdsi2010) [inline ivy/functional/frontends/jax/lax/operators.py:59]: add `out` parameter with default value `None` and then use the `out` parameter in the ivy.sin part

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_operators.py [hunk 2]
```diff
@@ -338,3 +338,34 @@ def test_jax_lax_acos(
...
+    num_positional_args,
+    native_array,
+    fw,
+):
+    input_dtype, x = dtype_and_x
+
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
```
Comment (ankitdsi2010) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_operators.py:364]: set `with_out` to True

**Review by ankitdsi2010 — CHANGES_REQUESTED** [review]

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_operators.py (hunk 2 above)
Comment (karthikgauti) [reply ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_operators.py:364]: I've just followed the tan operator which was already implemented in the main fork.  I've tried implementing the changes you requested, but I'm getting errors in Unit testing. Can you please check the Tan operator? 

def tan(x):
    return ivy.tan(x)


tan.unsupported_dtypes = {"torch": ("float16", "bfloat16")}`

**Comment by ankitdsi2010** [comment]
the tests aren't failing for your function, so you don't need to worry about it.

**Comment by ankitdsi2010** [comment]
Thanks!

**Pull request merged**